runtime/doc/tabpage.txt

```diff
@@ -176,8 +200,15 @@ REORDERING TAB PAGES:
 :[N]tabm[ove]
 		Move the current tab page to after tab page N.  Use zero to
 		make the current tab page the first one.  Without N the tab
-		page is made the last one.
-
+		page is made the last one. >
+		    :-tabmove   " move the tab page to the left
```

Comment:
OK,then, I'll review the suggestions and clean up. Thanks a lot for taking
a look 😉